tox.ini

```diff
@@ -23,36 +23,43 @@ commands=
 [testenv:lint]
 basepython=python3.9
 commands=
-  poetry run black --check --diff src/ tests/
-  poetry run flake8 src/ tests/
+    poetry run black --check --diff src/ tests/
+    poetry run flake8 src/ tests/
 
 [testenv:isort]
 basepython=python3.9
 commands=
-  poetry run isort --check-only --diff src/ tests/
+    poetry run isort --check-only --diff src/ tests/
+
+[testenv:slotscheck]
+basepython=python3.9
+commands_pre=
```

Comment:
defining `commands_pre` should not be needed. Isn't it inherited from the base testenv?

Reply:
So base **commands_pre** does not install project itself, but its required for slotscheck to properly import snug module https://github.com/ariebovenberg/snug/pull/367#issuecomment-1016992276

Reply:
correct, `poetry` does not install the project, but `tox` runs `pip install snug` internally 🤔.

Removing it worked "on my machine", so let's see what GH actions make of it